python/planout/experiment.py

```diff
@@ -330,3 +333,15 @@ def checksum(self):
         if not isinstance(src, six.binary_type):
             src = src.encode("ascii")
         return hashlib.sha1(src).hexdigest()[:8]
+
+class ProductionExperiment(SimpleExperiment):
```

Comment:
Can we make this a subclass of Experiment instead of SimpleExperiment?  SimpleExperiment is a nice way to get started for researchers running small experiments, but in any real prod situation, wouldn't want to use the added functionality of SimpleExperiment (which mostly has to do with logging).  All children of ProductionExperiment must therefore also implement their own the necessary abstract methods, e.g. log(), previously_logged().